demos/public/google541407f935830527.html

```diff
@@ -0,0 +1 @@
+google-site-verification: google541407f935830527.html
```

Comment:
I'm wondering how does the token here gets generated?

Reply:
The Google Search Console generates HTML files like these. It helps them verify someone owns a domain or URL path https://support.google.com/webmasters/answer/9008080?hl=en

Reply:
The HTML file upload method is the easiest verification method for AppEngine sites